data/remnant/remnant outfits.txt

```diff
@@ -874,6 +874,17 @@ outfit "Void Rifle"
 
 
 
+outfit "Void Sprite Parts"
+	category "Unique"
+	cost 500
+	thumbnail "outfit/mouthparts"
+	"mass" 1
+	"outfit space" -1
+	description "This is nothing more and nothing less than a cubic meter of void sprite parts. To the Remnant, this represents both a terrible occurrence and a dreadful opportunity."
+	description "	This could have been very valuable to someone who knew what they were doing, if it had been handled properly. Instead, it is worth little more than the value of the minerals and resources contained within."
```

Comment:
The Korath handled it 'less properly' then just cutting off a tentacle?

Reply:
This wasn't a careful harvest by people who wanted the void spirits parts for what they were, but rather the actions of people who merely wanted what was in them. Like harvesting whales for oil.